UVa/BasicRemains.java

```diff
@@ -0,0 +1,44 @@
+
+/**
+*Given a base b and two non-negative base b integers
```

Comment:
Nit: by the way, for the comment to really follow javadoc style it should be in this form:

    /**
      * <- all asterisks line up with the first asterisk
      * <- one space between an asterisk and the first character of a line
      */

Up to you if you want to make this change or not.